open-rosa/build.gradle.kts

```diff
@@ -0,0 +1,70 @@
+plugins {
```

Comment:
You created this new module that contains tests, but it looks like you forgot to let them run on CIrcleCI: https://github.com/getodk/collect/blob/master/docs/CODE-GUIDELINES.md#adding-a-new-module

Reply:
@seadowg What about this:

```kotlin
tasks.register("testDebug") {
  dependsOn("test")
}
```  

Is it still necessary, as the documentation suggests?

Reply:
@seadowg can't that new module be a java one?

Reply:
I don't think so without some rework. It uses Android classes like `MimeTypeMap`.